src/data/finsupp/fin.lean

```diff
@@ -0,0 +1,99 @@
+/-
+Copyright (c) 2021 Ivan Sadofschi Costa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ivan Sadofschi Costa
+-/
+import data.fin.tuple
+import data.finsupp.basic
+
```

Comment:
You should add some documentation at the beginning of the file. You can have a look at essentially any mathlib file to see an example and copy the style.

Reply:
Done!